egs/aishell/v1/local/split_data_enroll_match.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env python
+
+# Copyright 2017 Bengu Wu
+# Apache 2.0.
```

Comment:
I think it would be nice to leave a comment describing what this script is doing.